### ISSUE TITLE
[FirebaseCrashlytics] No issues in firebase console.

### ISSUE THREAD

**Issue by pgawlowski** [firebase/firebase-ios-sdk#4698]
**Describe your environment**

  * Xcode version: 11.3
  * Firebase SDK version: 6.25.0
  * Firebase Component: Crashlitics, Fabric
  * Component version: Crashlitics (3.12.0), Fabric (1.9.0) 

**Describe the problem**

After migration from Fabric to Firebase I encountered multiple problems with Crashlitics.
I followed this guide.
[(https://firebase.google.com/docs/crashlytics/get-started?fbclid=IwAR0YuotcIwobzk_3BOfQIKxVh-2129nlt1Z26pi5muzjZ2YkJq7ObwSMd0w)](url)
and used test implementation part to verify if it works fine. 

Effect is:
- crash-free statistics are going down as expected
- not seeing any crashes on dashboard list (invalid behaviour)

To sum up.
I am using debug format DWARF + dSYM.
I removed info.plist old "Fabric" section (not a single firebase + crashlitics tutorial mention this part). Only in that case Crashlitics.sharedInstance.debugMode = true seems to be working correctly.

Crashlitics logs are showing, inter alia:
> [Crashlytics:Crash] report submission successful
> [Answers] Finished transmission of '11XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXX46'.

So far it's been around 48h of struggling with that problem so, even assuming pretty long firebase delay, issues supposed to be there.

**Comment by rizafran** [comment]
Thanks for reporting the issue, @pgawlowski . May I know if you have removed your Fabric API key or made any changes in the setup? You may check out the docs [here](https://firebase.google.com/docs/crashlytics/migration-post-linking) for more details.

Also, could you try to undo removing of info.plist, and perform a [test crash on debug mode](https://firebase.google.com/docs/crashlytics/force-a-crash?platform=ios#enable_debug_mode) to see if you'll receive a "Report submission successful" message?

**Comment by pgawlowski** [comment]
Yes I removed Fabric API Key from info.plist. 
When it is present this part
`Crashlitics.sharedInstance.debugMode = true`

seems to have no effect at all - crashlytics logs are empty. Not a single message about sending, success or failure is present. What is more Firebase Console was not reporting any kind of effect. Stats are unaffected, issues are still empty.

**Comment by rizafran** [comment]
Hi @pgawlowski, as per docs(https://firebase.google.com/docs/crashlytics/migration-post-linking#what-happened), you shouldn’t remove the Fabric API key and not making changes on the setup. Please undo the changes you have made. If you’re still getting the same successful message but not receiving any reports, please file to Firebase Support (https://firebase.google.com/support) in order for them to help you fix the issue on receiving reports.

**Comment by pgawlowski** [comment]
@rizafran - read my last post. As soon as I reverted changes to info.plist Crashlytics is not responding at all. No logs at all. Nothing is being reported. Not a single log about Crashlytics.
Like I mentioned 
` Crashlytics.sharedInstance.debugMode = true`
this seems to have no effect at all as soon as I revert info.plist to original values.

**Comment by pgawlowski** [comment]
After discarding all changes and starting from scratch it seems to work fine now. No idea what was the initial problem. I still have no idea what was the initial problem with issues + Crashlytics still has some some logging problems - nothing is happening when "Fabric" key is present in info.plist.

**Comment by TKBurner** [comment]
Glad to hear you are up and running! :)
